crates/wiggle/test-helpers/src/borrow.rs

```diff
@@ -16,10 +12,6 @@ impl BorrowChecker {
     /// `GuestSlice` and `GuestStr` structs, which implement `std::ops::Deref` and
     /// `std::ops::DerefMut`. It also enforces that `GuestPtr::read` and `GuestPtr::write` do not
     /// access memory with an outstanding borrow.
-    /// The safety of this mechanism depends on creating exactly one `BorrowChecker` per
-    /// WebAssembly memory. There must be no other reads or writes of WebAssembly the memory by
-    /// either Rust or WebAssembly code while there are any outstanding borrows, as given by
-    /// `BorrowChecker::has_outstanding_borrows()`.
     pub unsafe fn new() -> Self {
```

Comment:
Technically this can probably be safe now since the unsafety is now in the `unsafe impl GuestMemory for ...`

Reply:
Thanks! Fixed in 5f5d7cb, will merge once CI passes